unstructured/partition/pdf.py

```diff
@@ -553,6 +524,7 @@ def _process_pdfminer_pages(
     fp: BinaryIO,
     filename: str = "",
     include_page_breaks: bool = False,
+    languages: Optional[List[str]] = None,
```

Comment:
I tried removing the default, but then I was getting an linting error about having a non-default argument coming after a default argument. I was unsure about changing the order of the arguments to address that since I have been told to not change the order of the arguments in case users are using positional arguments. Thinking about it, though, that shouldn't be the case for private functions, so I will change it if all of that sounds correct to you.